core/client/app/components/gh-uploader.js

```diff
@@ -28,14 +28,14 @@ var PostImageUploader = Ember.Component.extend({
         $this.on('imagecleared', function () {
             self.sendAction('canceled');
         });
-    }.on('didInsertElement'),
+    },
 
-    removeListeners: function () {
+    willDestroyElement: function () {
```

Comment:
Or if not that, there's these things called comments that our client code is rather lacking